.github/workflows/check-pr-title.yml

```
@@ -0,0 +1,21 @@
+name: Check PR Title
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+
+jobs:
+  check-title:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check PR title
+      run: |
+        title="${{ github.event.pull_request.title }}"
+        if ! echo "$title" | grep -qE '^(build|chore|ci|docs|style|refactor|perf|test|feat|fix): '; then
```

```suggestion
        if ! echo "$title" | grep -qE '^(build|chore|ci|docs|feat|fix|perf|refactor|revert|style|test)(\(\w+\))?!?: '; then
```
Adds:
- `revert` (from committed.toml)
- optional scope: `(scope)`
- optional `!` to indicate breaking changes